# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../units/gcc33-support.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- units-1.80/units.c~	2002-05-07 00:20:38.000000000 +0200
+++ units-1.80/units.c	2003-08-12 23:47:18.000000000 +0200
@@ -2358,7 +2358,7 @@
 
 #if 0
 
-    "sed 's/.*/& ft\"
+    "sed 's/.*/& ft/'"
     "m/' feet|units -q|sed 's/^[[:space:]]*\* //"
     "/^[[:space:]]*\//d'"
 
